public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -67,8 +72,9 @@ export default class AlertsDashboardFlyoutComponent extends Component {
       alerts: [],
       alertState: alertState,
       loading: true,
-      monitors: [],
+      monitor: monitor,
```

Comment:
We are now changing this from a list to a single value. Are there concerns for this?

Reply:
Not for the alerts flyout. It's intended to only display alerts for a single trigger; and that trigger is only associated with 1 monitor.